src/cpp/fastdds/xtypes/type_representation/TypeObjectRegistry.cpp

```diff
@@ -338,6 +338,20 @@ ReturnCode_t TypeObjectRegistry::get_type_information(
     return ret_code;
 }
 
+ReturnCode_t TypeObjectRegistry::get_direct_hash_type_dependencies(
```

Comment:
I suggest that this method keeps the old name and the internal method is the one to change the name. For example, the internal could be called `get_type_dependencies_impl`.

Reply:
moved in [d3245e1](https://github.com/eProsima/Fast-DDS/pull/4496/commits/d3245e16dd687c62ce397be8018c718828d2bc6e)